.gitignore

```diff
@@ -21,3 +21,6 @@ dist/
 
 # PyCharm
 .idea
+
```

Comment:
please delete this changes. if you make this changes please create new pr for this.

Reply:
> please delete this changes. if you make this changes please create new pr for this.

Done